apps/simple-trading-app/src/app/components/deal-ticket/deal-ticket-container.tsx

```diff
@@ -1,18 +1,23 @@
-import type { DealTicketContainerProps } from '@vegaprotocol/deal-ticket';
 import {
   DealTicketManager,
   DealTicketContainer as Container,
 } from '@vegaprotocol/deal-ticket';
 import { DealTicketSteps } from './deal-ticket-steps';
+import { useParams } from 'react-router-dom';
 
-export const DealTicketContainer = ({ marketId }: DealTicketContainerProps) => {
-  return (
+const tempEmptyText = <p>Please select a market from the markets page</p>;
```

Comment:
non-blocking: we can use here a `t` helper, as it will, in the future, contain a solution for language versions

Reply:
Sure :) This one is temp and will not be used once we get add the select markets feature in trading view